backend/src/components/lookup.js

```diff
@@ -55,31 +55,32 @@ const fundingModelType = [
   },
 ];
 
-function parseProgramYear(value) {
+function parseProgramYear(programYearList) {
```

Comment:
Would it be cleaner to have a single getProgamYear() function (like getLicenseCategories() without the extra cache) then having a call getOperation and then parseProgramYear()? It would make getLookupInfo() cleaner.

Reply:
My suggestion was to put all the logic in a getProgramYear() function so getLookupInfo() just calls it.

Reply:
sorry I misread this!